src/Rector/Deprecation/AssertNoUniqueTextRector.php

```diff
@@ -17,11 +17,6 @@ final class AssertNoUniqueTextRector extends AbstractRector
 
     use GetDeclaringSourceTrait;
 
-    /**
-     * @todo remove when property is no longer private in AbstractRector.
-     */
-    private $nodesToAddCollector;
-
     public function __construct(
         NodesToAddCollector $nodesToAddCollector
     ) {
```

Comment:
We could probably drop this constructor if it is fixed upstream? Need to review full code

Reply:
Hi @mglaman What do you mean by fixed upstream?

Reply:
@iyyappandrupal upstream being the class in Rector. If it works with the constructor removed, the PR should be updated to remove it (cc @FlorentTorregrosa or @agentrickard) in both classes. That way we have less code to maintain.

Reply:
I am on it

Reply:
@mglaman Do you mean AbstractRector class in Rector library?Thanks

Reply:
PR updated. But warning, I have not tested locally as I was not on my standard dev environment.

Reply:
@FlorentTorregrosa I tested morning in my project after removing the constructor, it works without well.